templates/taxi/manufacturer_list.html

```diff
@@ -8,6 +9,10 @@ <h1>
     </a>
   </h1>
 
+  <form action="" method="get" class="form-inline">
+    {{ search_form|crispy }}
+    <input type="Submit" class="btn btn-secondary" value="🔍">
+  </form>
```

Comment:
Please create a separate template for the search form and include it where needed.